documentation/src/docs/controller-service.md

```diff
@@ -47,39 +47,35 @@ You may obtain a copy of the License at
+The Controller Service is a core component of Pravega that implements
 the control plane. It acts as the central coordinator and manager for
-various operations performed in the cluster, mainly divided into two
-categories: **Stream Management** and **Cluster Management**.
-
-The Controller service, referred to simply as *Controller* henceforth, is
-responsible for providing the abstraction of a [Stream](pravega-concepts.md#streams), which is the
-main abstraction that Pravega exposes to applications. A stream
-comprises one or more [segments](pravega-concepts.md#stream-segments). Each segment is an append-only data
-structure that stores a sequence of bytes. A segment on its own is
-agnostic to presence of other segments and is not aware of its logical
-relationship with its peer segments. The Segment Store, which owns and
-manages these segments, does not have any notion of a stream. A stream
+various operations performed in the cluster, the major two
+categories are: **Stream Management** and **Cluster Management**.
+
+The Controller Service, referred to simply as **Controller** henceforth, is
+responsible for providing the abstraction of a Pravega [Stream](pravega-concepts.md#streams), which is the main abstraction that Pravega exposes to applications. A Pravega Stream
+comprises one or more Stream [Segments](pravega-concepts.md#stream-segments). Each Stream Segment is an append-only data
+structure that stores a sequence of bytes. A Stream Segment on its own is
+agnostic to the presence of other Stream Segments and is not aware of its logical
+relationship with its peer Stream Segments. The Segment Store, which owns and
+manages these Stream Segments, does not have any notion of a Stream. A Stream
 is a logical view conceptualized by Controller by composing a
-dynamically changing set of segments that satisfy a predefined set of
-logical invariants. The Controller provides the stream abstraction and
-orchestrates all lifecycle operations on a stream while ensuring that
+dynamically changing set of Stream Segments that satisfy a predefined set of
+logical invariants. The Controller provides the Stream abstraction and
+orchestrates all lifecycle operations on a Pravega Stream while ensuring that
 the abstraction stays consistent.
 
-The Controller plays a central role in the lifecycle of a stream:
+The Controller plays a central role in the lifecycle of a Stream:
 _creation_, _modification_, [_scaling_](pravega-concepts.md#elastic-streams-auto-scaling), and _deletion_. It does these by
-maintaining metadata per stream and performs requisite operations on
-segments as and when necessary. For example, as part of stream’s
-lifecycle, new segments can be created and existing segments sealed. The
-Controller decides when to perform these operation such that streams
-continue to be available and consistent to the clients accessing them.
+maintaining metadata per Stream and performs requisite operations on
```

Comment:
IMHO, `requisite operations` sounds odd. Maybe we can say: `It does these by...` -> To implement these operations, the Controller manages both a Stream's metadata and its associated Stream Segments.

Reply:
Accepted suggestion